test/cookbooks/sensu_test/recipes/default.rb

```diff
@@ -111,3 +111,8 @@
   labels(environment: 'production', region: 'us-west-2')
   annotations(runbook: 'https://www.xkcd.com/378/')
 end
+
+sensu_hook 'cron' do
```

Comment:
I think it would be better to rename this to be a bit more clear as to its intent if your inspecting it via the resource collection. How about something like this:
```suggestion
sensu_hook 'restart_cron_service' do
```
or
```suggestion
sensu_hook 'cron_service_restart' do
```